src/main/scala/com/typesafe/conductr/client/ConductRController.scala

```diff
@@ -192,6 +210,13 @@ object ConductRController {
         )
       )
     )
+
+  private def extractZipEntry(entryName: String, zipFile: Uri, destDir: File): Uri = {
+    import sbt.NameFilter._
+    val result = IO.unzip(new File(absolute(zipFile).path.toString()), destDir, { fileName: String => fileName.contains(entryName) }).head
+    Uri(result.getAbsolutePath)
+  }
+
```

Comment:
Also we have a similar logic already in the `ConductR.scala` file in which we unzip the bundle as well and access the `bundle.conf` file: https://github.com/fsat/sbt-conductr/blob/ctrl-protocol-v2/src/main/scala/com/typesafe/conductr/sbt/ConductR.scala#L41-L45

Maybe this logic can be combined?